### ISSUE TITLE
What is the CSS class for selected tabs?

### ISSUE THREAD

**Issue by Evotron** [piroor/treestyletab#2840]
I′m using dark mode on Firefox. When I select tabs, as in, for example, when I select all tabs, they hardly change color. It′s hard for me to tell when I am selecting a group of tabs because the color is so similar to the default color, especially because I′m using night light to get rid of blue light.

What′s the CSS class for selected tabs? Is there some way for me to figure this out on my own?

Here is a picture. I have selected all the Wikipedia tabs, but I want to change the color and style of the selected tabs.
![selected tabs cannot see 1](https://user-images.githubusercontent.com/46901105/112251649-abdf0680-8c29-11eb-98b7-ac4379577f42.jpg)

**Comment by irvinm** [comment]
https://github.com/piroor/treestyletab/wiki/How-to-inspect-tree-of-tabs

**Comment by irvinm** [comment]
You may also want to look over https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules for ideas.

**Comment by Evotron** [comment]
Many thanks.

It seems the exact instructions are outdated?! Here′s what I did:

1. Type about:debugging into the location bar and hit the Enter key.
2. Click on “This Firefox”
3. Find Tree Style Tab and click on its corresponding “Inspect” link

There I was able to eventually find the styling by looking through the various style sheets and searching for the term “selected”.

Here is what I came up with for styling:

```
/* change color of selected tabs */
:root {
  --multiselected-color: #3FFF00;
  --multiselected-color-opacity: .5;
}
```

Added that to treestyletab.css in options, and I was set.
![selected tabs new style 1](https://user-images.githubusercontent.com/46901105/112367727-4b42de80-8ca8-11eb-9484-de5c446124d3.jpg)
